ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.cpp

```diff
@@ -121,28 +144,59 @@ void BalovnevModelPlugin::getForces(double VERTICAL_CUT_DEPTH)
   double q    = SURCHARGE_MASS; 
   double h    = SOIL_PRISM_HEIGHT;
   double v    = BUCKET_VELOCITY;
-  // define VERTICAL_CUT_DEPTH
-  double d = VERTICAL_CUT_DEPTH;
+  // define vertical_cut_depth
+  double d = vertical_cut_depth;
   // get constant a1,a2,a3
-  double a1 = getParameterA ( beta, phi, delta );
-  double a2 = getParameterA ( ea, phi, delta );
-  double a3 = getParameterA ( M_PI_2, phi, delta );
+  double a1 = getParameterA(beta, phi, delta);
+  double a2 = getParameterA(ea, phi, delta);
+  double a3 = getParameterA(M_PI_2, phi, delta);
   //calculate horizontal force and vertical force
-  m_horizontal_force = w*d*a1 * (1+(1/tan(beta))*tan(delta)) * (d*g*sd/2 + c*(1/tan(phi))
-                      + g*q + BURIED * (d-l*sin(beta)) * g*sd * (1-sin(phi)) / (1+sin(phi)))
-                      + w*et*a2 * (1 + tan(delta)*(1/tan(ea))) * (et*g*sd/2 + c*(1/tan(phi))
-                      + g*q + d*g*sd * (1-sin(phi))/(1+sin(phi)))
-                      + d*a3 * (2*s + 4*ls*tan(delta))*(d*g*sd/2 + c*(1/tan(phi))
-                      + g*q + BURIED * (d-ls*sin(beta)) * g*sd * (1-sin(phi))/(1+sin(phi)));
+  m_horizontal_force =
+    w*d*a1 * (1+(1/tan(beta))*tan(delta)) * (d*g*sd/2 + c*(1/tan(phi))
+    + g*q + BURIED * (d-l*sin(beta)) * g*sd * (1-sin(phi)) / (1+sin(phi)))
+    + w*et*a2 * (1 + tan(delta)*(1/tan(ea))) * (et*g*sd/2 + c*(1/tan(phi))
+    + g*q + d*g*sd * (1-sin(phi))/(1+sin(phi)))
+    + d*a3 * (2*s + 4*ls*tan(delta))*(d*g*sd/2 + c*(1/tan(phi))
+    + g*q + BURIED * (d-ls*sin(beta)) * g*sd * (1-sin(phi))/(1+sin(phi)));
```

Comment:
Low priority, but this could be shorter and perhaps easier to follow if meaningful sub-computations and repeated ones (e.g. `1+sin(phi)) were given names.